server/utils/colors.js

```diff
@@ -64,9 +64,68 @@ function hexToInt(hexColor) {
   return parseInt(hexColor, 16);
 }
 
+/**
+ * @description Converts XY color (CIE 1931 color space) to int.
+ * @param {number} x - X color.
+ * @param {number} y - Y color.
+ * @returns {number} Int color.
+ * @example
+ * const int = xyToInt(0.701, 0.299);
+ * console.log(int === 16711680);
+ */
+function xyToInt(x, y) {
+  /**
+   * @description Reverse Gamma correction applied in rgbToXy.
+   * @param {number} value - Color value.
+   * @returns {number} ReversedGammaCorrectedValue - Color with Gamma added.
+   * @example
+   * const value = getReversedGammaCorrectedValue(0,5);
+   * console.log(value === 0.73535698305);
+   */
+  function getReversedGammaCorrectedValue(value) {
```

Comment:
Maybe this method can out of `xyToInt`, as a const one?

Reply:
Like it's very dedicated to the `xyToInt` function, I preferred to keep it inside (not usable outside the function).
Do you think it's better to put it outside ?

Reply:
It's to avoid to declare same function each time we enter in upper method.

Reply:
+1, as this function doesn't use variable from xyToInt, it doesn't need to be re-created at each call to xyToInt. It's better to create it outside!

Reply:
Fixed